src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -303,8 +303,8 @@ static Option firstVariadicArg(int index) {
          *          before it can be overwritten by the Java runtime, or read through conventional means}
          * <p>
          * A downcall method handle linked with this option will feature an additional {@link MemorySegment}
-         * parameter directly following the target address parameter. This memory segment must be a
-         * native segment into which the captured state is written.
+         * parameter directly following the target address, and optional SegmentAllocator parameters.
```

Comment:
use `{@code}` or `{@link}` for SegmentAllocator

Reply:
Ah, right. Updated